www/src/pages/foundations/colors.jsx

```diff
@@ -84,7 +84,7 @@ const renderColorRamp = (themeName, unusedLevels) => (
     key={`${themeName}`}
     style={{ flexBasis: '24%', marginRight: '1%', marginBottom: '2rem' }}
   >
-    <h5>{themeName}</h5>
+    <h2>{themeName}</h2>
```

Comment:
Let's keep the `h5` styling, while using the `h2` element via the `h5` class name.